code/modules/food/customizables.dm

```diff
@@ -78,67 +78,72 @@
 	var/fullyCustom = 0
 	var/addTop = 0
 
-
 /obj/item/weapon/reagent_containers/food/snacks/customizable/New(loc,ingredient)
 	. = ..()
-	src.reagents.add_reagent("nutriment",8)
+	src.reagents.add_reagent("nutriment",3)
 	src.update()
 	return
 
-/obj/item/weapon/reagent_containers/food/snacks/customizable/attackby(obj/item/I,mob/user)
+/obj/item/weapon/reagent_containers/food/snacks/customizable/attackby(obj/item/I, mob/user, params)
 	if((src.contents.len >= src.ingMax) || (src.contents.len >= ingredientLimit))
-		user << "<span class='warning'>How about no.</span>"
+		user << "<span class='warning'>That's already looking pretty stuffed.</span>"
 	else if(istype(I,/obj/item/weapon/reagent_containers/food/snacks))
 		var/obj/item/weapon/reagent_containers/food/snacks/S = I
+		S.reagents.trans_to(src,S.reagents.total_volume)
 		user.drop_item(I, src)
 		src.ingredients += S
-		S.reagents.trans_to(src,S.reagents.total_volume)
+
+		var/clicked_x = text2num(params2list(params)["icon-x"])
+		if (isnull(clicked_x))   S.pixel_x = 0
+		else if (clicked_x < 9)  S.pixel_x = -2 //this looks pretty shitty
+		else if (clicked_x < 14) S.pixel_x = -1 //but hey
+		else if (clicked_x < 19) S.pixel_x = 0  //it works
+		else if (clicked_x < 25) S.pixel_x = 1
+		else S.pixel_x = 2
+
 		src.update()
 		user << "<span class='notice'>You add the [I.name] to the [src.name].</span>"
-	else . = ..()
-	return
+	else
+		return ..()
 
 /obj/item/weapon/reagent_containers/food/snacks/customizable/proc/update()
 	var/i = 1
 	var/image/I
 	var/new_name
 	src.overlays.len = 0
+	/*The overlays have to be re-drawn every time the foodstuff is updated, this is because there's no reliable way to
```

Comment:
There IS!
You should store the topping in a var, which you can constantly ref to remove and readd the overlay.
It's a lot cheaper.